server/routes/messages.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+const Pusher = require('pusher');
+
+
+const pusher = new Pusher({
+  appId: 'xxxxxx',
```

Comment:
This probably isn't the best way to deal with this. You should be getting these value from your process.env / .env file.